x-pack/elastic-agent/pkg/agent/program/testdata/single_config.yml

```diff
@@ -25,145 +25,151 @@ outputs:
-      hosts: ["http://apache.remote"]
-  hosts: ["http://apache.local"]
-- type: synthetics/http
-  id: unique-http-id
-  name: my-http
-  schedule: '*/5 * * * * * *'
-  host: "http://localhost:80/service/status"
-  timeout: 16s
-  wait: 1s
-- id: endpoint-id
-  type: endpoint
-  name: endpoint-1
-  enabled: true
-  package:
-    name: endpoint
-    version: 0.3.0
-  data_stream:
-    namespace: default
-  artifact_manifest:
-    schema_version: v22
-    manifest_version: v21
-    artifacts:
-      - endpoint-allowlist-windows:
-          sha256: 1234
-          size: 2
-          url: /relative/path/to/endpoint-allowlist-windows
-      - endpoint-allowlist-macos:
-          sha256: 1234
-          size: 2
-          url: /relative/path/to/endpoint-allowlist-macos
-      - endpoint-allowlist-linux:
-          sha256: 1234
-          size: 2
-          url: /relative/path/to/endpoint-allowlist-linux
-  policy:
-    linux:
-      advanced:
-        free-form: free-form-value
-      indices:
-        network: logs-endpoint.events.network-default
-        file: logs-endpoint.events.file-default
-        process: logs-endpoint.events.process-default
-        metadata: metrics-endpoint.metadata-default
-        policy: metrics-endpoint.policy-default
-        telemetry: metrics-endpoint.telemetry-default
-      logging:
-        file: info
-        stdout: debug
-      events:
-        process: true
-        file: true
-        network: true
-    windows:
-      malware:
-        mode: prevent
-      advanced:
-        free-form: free-form-value
-      indices:
-        network: logs-endpoint.events.network-default
-        file: logs-endpoint.events.file-default
-        registry: logs-endpoint.events.registry-default
-        process: logs-endpoint.events.process-default
-        driver: logs-endpoint.events.driver-default
-        library: logs-endpoint.events.library-default
-        alerts: logs-endpoint.alerts-default
-        metadata: metrics-endpoint.metadata-default
-        policy: metrics-endpoint.policy-default
-        telemetry: metrics-endpoint.telemetry-default
-      logging:
-        file: info
-        stdout: debug
-      events:
-        registry: true
-        process: true
-        security: true
-        file: true
-        dns: false
-        dll_and_driver_load: false
-        network: true
-    mac:
-      malware:
-        mode: prevent
-      advanced:
-        free-form: free-form-value
-      indices:
-        network: logs-endpoint.events.network-default
-        file: logs-endpoint.events.file-default
-        process: logs-endpoint.events.process-default
-        alerts: logs-endpoint.alerts-default
-        metadata: metrics-endpoint.metadata-default
-        policy: metrics-endpoint.policy-default
-        telemetry: metrics-endpoint.telemetry-default
-      logging:
-        file: info
-        stdout: debug
-      events:
-        process: true
-        file: true
-        network: true
+  - type: docker/metrics
```

Comment:
We should probably "standardise" on the indentation to not have too big diffs here in the future. Linting rule perhaps?